.github/actions/dbp-charts/verify-compose/action.yml

```diff
@@ -25,10 +25,6 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Checkout
-      uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
-      with:
-        fetch-depth: 0
```

Comment:
if there was fetch-depth then it should had something to do with parsing commits, but I don't see anymore which could rely on that so ok to drop it